src/apps/vpn/mozillavpn.cpp

```diff
@@ -910,7 +910,7 @@ void MozillaVPN::logout() {
 void MozillaVPN::reset(bool forceInitialState) {
   logger.debug() << "Cleaning up all";
 
-  TaskScheduler::deleteTasks();
```

Comment:
I agree with the call to `MozillaVPN::deactivate()` but why remove the call to `TaskScheduler::deleteTasks()`?

Reply:
Beucase that is done inside `deactivate` already.

Reply:
Perfect! Approved!